src/main/java/taxi/dao/jdbc/ManufacturerJdbcDao.java

```diff
@@ -0,0 +1,112 @@
+package taxi.dao.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import taxi.dao.ManufacturerDao;
+import taxi.lib.Dao;
+import taxi.lib.Inject;
+import taxi.model.Manufacturer;
+import taxi.services.util.ConnectionService;
+
+@Dao
+public class ManufacturerJdbcDao implements ManufacturerDao {
+    @Inject
+    ConnectionService connection;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "insert into manufacturers (name, country) values (?,?)";
```

Comment:
```suggestion
        String createQuery = "INSERT INTO manufacturers (name, country) VALUES (?,?)";
```
для ключевых слов запроса лучше использовать заглавные, чтоб была лучше читаемость (для остальных тоже самое), более информативное название для запросов (query)

Reply:
fixed